### DISCUSSION TITLE
Is 5.13.0 released ?

### DISCUSSION THREAD

**Discussion by Alansyf** [noobaa/noobaa-operator#1160] [General]
### Environment info

`VERSION=$(curl -s https://api.github.com/repos/noobaa/noobaa-operator/releases/latest | jq -r '.name')
`

```
echo @VERSION 
v5.13.0
```

Then curl -LO https://github.com/noobaa/noobaa-operator/releases/download/$VERSION/noobaa-$OS-$VERSION

But latest noobaa mac CLI is 5.11.0

**Comment by nimrod-becker** [comment]
We had a technical problem with publishing the artifacts, we hope to have it all sorted out today
@utkarsh-pro

**Comment by tangledbytes** [comment]
Hi, I think the changed packaging format might be throwing everyone off. Instead of publishing binaries directly now we publish tarballs for multiple platforms. This works for me on mac:

```
$ wget 'https://github.com/noobaa/noobaa-operator/releases/download/v5.13.0/noobaa-operator-v5.13.0-darwin-arm64.tar.gz'
$ tar -xvzf noobaa-operator-v5.13.0-darwin-arm64.tar.gz
$ ./noobaa-operator version
INFO[0000] CLI version: 5.13.0                          
INFO[0000] noobaa-image: noobaa/noobaa-core:master-20230123 
INFO[0000] operator-image: noobaa/noobaa-operator:5.13.0
```

**Reply by tangledbytes** [reply]
Ah, I see where you are coming from. README does not reflect this change right now. Will update the README.

**Reply by Alansyf** [reply]
I use https://github.com/noobaa/noobaa-operator/releases/download/v5.13.0/noobaa-operator-v5.13.0-darwin-amd64.tar.gz instead but when start it: 

`Failed to pull image "noobaa/noobaa-operator:5.13.0": rpc error: code = Unknown desc = Error response from daemon: manifest for noobaa/noobaa-operator:5.13.0 not found: manifest unknown: manifest unknown`

**Reply by tangledbytes** [reply]
@Alansyf just pushed the images. Please try running again now.
